partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+    count_inflight_subprocess = 0
+
+    def __init__(self, ctx, source_project, target_project, config):
+        """
+        Initialize the OBJ part factory.
+        """
+        with pc_logging.Action("InitOBJ", target_project.name, config["name"]):
+            super().__init__(ctx, source_project, target_project, config, extension=".obj")
+            self._create(config)
+            self.runtime = None  # Lazy initialization for subprocess runtime
+
+    async def instantiate(self, part):
```

Ruff (0.8.2)

Missing return type annotation for public function `instantiate`

(ANN201)